src/guides/writing-skills/sending-messages/index.md

```diff
@@ -0,0 +1,298 @@
+---
+uid: sending-messages
+---
+
+# Sending messages
+
+One of the most important things your skill needs to do is to communicate with users.
+The easiest way to send a message is to call the `Reply` API
+
+## [C#](#tab/tabid-cs)
+
+```csharp
+await Bot.ReplyAsync("Thanks!");
+```
+
+## [JavaScript](#tab/tabid-js)
+
+```js
+bot.reply("Thanks!");
```

Comment:
Best to check with @pmn and @shana on this but I think it's best to await the reply. I seem to remember getting out of order replies when I didn't. But I never created a clear repro case.

```suggestion
await bot.reply("Thanks!");
```

Reply:
Yes yes, these should be awaited. It doesn't matter if they're not awaited when they're the last instruction to be executed, but as a best practices/examples doc, definitely encouraging awaits is a good thing imo

Reply:
Yep, absolutely agree.